### ISSUE TITLE
When "bigchaindb start", give user hints on what they can do next

### ISSUE THREAD

**Issue by ttmc** [bigchaindb/bigchaindb#55]
When you start RethinkDB, it gives some hinds on what you can do next:
```text
Listening for intracluster connections on port 29015
Listening for client driver connections on port 28015
Listening for administrative HTTP connections on port 8080
Listening on addresses: 127.0.0.1, 127.0.1.1, ::1
To fully expose RethinkDB on the network, bind to all addresses by running rethinkdb with the `--bind all` command line option.
Server ready, "ascribe3_XPS_8700_5nu" fafe8d60-0494-4ea9-ad78-5305b6fc97b5
```

Brett suggested we could have some hints like that when the user runs `bigchaindb start`. Here's what it says now:
```text
Generating keypair
Database host? (default `localhost`): 
Database port? (default `28015`): 
Database name? (default `bigchain`): 
Ready to go!
INFO:bigchaindb.config_utils:Configuration loaded from `/home/troy/.bigchaindb`
INFO:bigchaindb.db.utils:Create:
INFO:bigchaindb.db.utils: - database `bigchain`
INFO:bigchaindb.db.utils: - tables
INFO:bigchaindb.db.utils: - indexes
INFO:bigchaindb.db.utils: - genesis block
INFO:bigchaindb.db.utils:Done, have fun!
INFO:bigchaindb.commands.bigchain:Start bigchaindb main process
INFO:bigchaindb.processes:starting bigchain mapper
INFO:bigchaindb.processes:starting backlog mapper
INFO:bigchaindb.processes:starting block
INFO:bigchaindb.processes:starting voter
INFO:bigchaindb.block:bootstraping block module...
INFO:bigchaindb.block:finished reading past transactions
INFO:bigchaindb.voter:voter waiting for new blocks
INFO:bigchaindb.block:finished bootstraping block module...
INFO:bigchaindb.block:starting block module...
```

That is fairly informative, but then it feels like it maybe got stuck. Is it still starting block module? Is it "ready" in some sense? If so, in what sense?

**Comment by Mec-iS** [comment]
> Listening for intracluster connections on port 29015
> Listening for client driver connections on port 28015
> Listening for administrative HTTP connections on port 8080
> Listening on addresses: 127.0.0.1, 127.0.1.1, ::1
> To fully expose RethinkDB on the network, bind to all addresses by running rethinkdb with the `--bind all` command line option.
> Server ready, "ascribe3_XPS_8700_5nu" fafe8d60-0494-4ea9-ad78-5305b6fc97b5

The lines are well informative, usually in Linux this means "process is running". I think just add "Process is running at PID xyzw" (the outcome from `pgrep bigchaindb`?) before "Server Ready" could be fine.

I usually run the command with: `nohup bigchaindb start &`, it prints back one of the PIDs.
This actually makes much clearer the thing and you get the command line usable again but I got some problems at stopping the background preocesses (as it starts 13 different processes). Having something like `bigchaindb stop` to kill all the processes would be great.

Addendum: it would be nice to have the precise command to run the `conftest.py` and other tests files added after the benchmark in the `README`.

**Comment by r-marques** [comment]
A more descriptive initialization message is resolved in #139 
Document how to run the tests will be resolved in #140 
There is an open issue for a clean exit of all bigchaindb processes #122
